include/cpu/i386/cpu.h

```diff
@@ -113,22 +125,13 @@ typedef struct CPUX86State {
     uint16_t fpus;
```

Comment:
Do you need to move these too? If not, could write a quick comment in the code

Reply:
In the first version I moved them too, but later I moved them out to make debugging easier, maybe should moved too in the final version.